### DISCUSSION TITLE
General Workflow for someone new to photo management

### DISCUSSION THREAD

**Discussion by Rigel09** [immich-app/immich#6098] [Q&A]
Hello all

First of all thanks for such an amazing app. 

I'm making this post to ask about a good workflow. I'm wanting to consolidate all of our family's photos (not much 20Gb) and make them more easy to access. I have never used a photo management app in my life (including app-which-must-not-be-named). For my entire life they have just lived in a photos folder on the root drive sorted like so. Viewing is done via what ever the default photo app is inside the file explorer.

```
-photos
     -year
          -album
```

I know I can recreate that workflow using the [storage templates](https://immich.app/docs/install/post-install/#step-3---update-the-storage-template), however I am confused on the upload photo / external libraries implementation.

My main question is can two users have the same upload location that would be specified under `Account->Account Settings->Storage Label`?

I would like all of our photos to live in some top level directory that isn't segmented by user since most of the photos belong to the both of us and all should be viewable by both of us. This is the reason for asking if two users can share the same upload directory. However, its also possible that my future workflow could be wrong. Perhaps each user should have different upload locations and then after upload they get moved to a Family library? Is this a good idea? I couldn't find anything related to this in the documentation other than [partner sharing](https://immich.app/docs/features/partner-sharing) which doesn't seem to be what I am looking for. How would this handle both of us having the same album (june-vacation) for instance. The photos in this album would be split between two locations? My mindset for this is something like this is similar to 'groups' in the Linux file system. Anyone with access to the Family library can have read and write permissions to that library. Maybe another user only has read permissions? My main reason for asking this is regards to backup. When I create the backup I would like to have the ability to easily find the photo I am looking for without using Immich.

I see that a user can create a new [external library](https://immich.app/docs/features/libraries) but that is read only, photos can't be added to it unless outside the app. I was originally waiting on this feature like a lot of other people before trying Immich out because I thought that the photos were stored in a database and not on a file system. However I can find the photos that I upload in `upload-dir/library/user/` so now I'm confused on why so many people were waiting for this feature as the JPEG is in a plain file folder structure?

I've tried search here and on Reddit for good workflows that other people have shared but haven't found anything that seems to satisfy my question. I posted here as I don't have a Reddit account and would rather not make one. I hope this is a suitable place to ask.

TLDR
What is the workflow you use in Immich where photos in an album are split between users?

Thanks for any help / advice and I hope everyone is having a happy new year,
Rigel

**Comment by XMMM1** [comment]
If i may add regarding to opened topic...

Personally i am ok with mentioned external library. But the case is this folder have a ton of unsorted images waiting for someone to get them sorted... but some of them are already in some folders waiting to be placed in hierarchically proper order. For this case i found very useful [folder view](https://immich.app/docs/features/libraries#folder-view). But i am a bit confuzed as this feature do not offer anything except folder view of external lib. 

I am not sure if i am missing something as i initially want to use this view for image movement to proper folder specific folder sharing etc.. 

The question is how is organizing images ment for external libs? (sorted unsorted images; keeping folder structure of organized folders)
